src/eddymotion/model/base.py

```diff
@@ -448,6 +449,69 @@ class DKIModel(BaseModel):
     _model_class = "dipy.reconst.dki.DiffusionKurtosisModel"
 
 
+class GaussianProcessModel:
```

Comment:
Do we want to inherit from model base?
```suggestion
class GaussianProcessModel(ModelBase):
```

Reply:
duh, we need to merge #176

Reply:
PR is merged. Done in 10b4a6b.